File: tecnicaturagit3Py/Leccion00/Intro.py
class IntegrantesFunkTheCode2023:
  
    def __init__(self):
        self.nombres_apellidos = 
        [
            "Nicolás Rizo Avellaneda",
            "Andrés Montes",
            "Danilo Profita",
            "María Bárbara Natasha Herrera",
            "Sebastián Bustamante",
            "Carla Fuschino",
            "Matias Diaz de Otazu",
            "Catriel Quintana"
        ]

    def imprimir_integrantes(self):
        for nombre_apellido in self.nombres_apellidos:
          
            nombre, apellido = nombre_apellido.split()
          
            print("Nombre:", nombre)
          
            print("Apellido:", apellido)
          
            print("")

integrantes = IntegrantesFunkTheCode2023()

integrantes.imprimir_integrantes()

File: Java/Leccion00/Intro.java
public class IntegrantesFunkTheCode2023 {
  
    private String[] nombresApellidos = 
    {
        "Nicolás Rizo Avellaneda",
        "Andrés Montes",
        "Danilo Profita",
        "María Bárbara Natasha Herrera",
        "Sebastián Bustamante",
        "Carla Fuschino",
        "Matias Diaz de Otazu",
        "Catriel Quintana"
    };

    public void imprimirIntegrantes() {
      
        for (String nombreApellido : nombresApellidos) 
          {
            String[] partes = nombreApellido.split(" ");
            String nombre = partes[0];
            String apellido = partes[1];
            System.out.println("Nombre: " + nombre);
            System.out.println("Apellido: " + apellido);
            System.out.println();
        }
    }

    public static void main(String[] args) {
        IntegrantesFunkTheCode2023 integrantes = new IntegrantesFunkTheCode2023();
      
        integrantes.imprimirIntegrantes();
    }
}
